pkg/os/launchd/launchd_darwin.go

```diff
@@ -112,12 +114,12 @@ func CheckPlist(config AgentConfig) error {
 
 // LoadPlist loads a launchd agents' plist file
 func LoadPlist(label string) error {
-	return runLaunchCtl("load", getPlistPath(label))
+	return runLaunchCtl("load", "-w", getPlistPath(label))
```

Comment:
aha, does the `-w` mean its going to wait till the operation returns?

Reply:
No as per launchctl man page this suppose to enable the service 
```
              -w       Overrides the Disabled key and sets it to false or true for the load and unload subcommands respectively. In previous versions, this option would modify the
                       configuration file. Now the state of the Disabled key is stored elsewhere on- disk in a location that may not be directly manipulated by any process other than
                       launchd.

```